CMakeLists.txt

```diff
@@ -598,6 +598,7 @@ endif()
 
 ## Set RCCL compile options
 target_compile_options(rccl PRIVATE -parallel-jobs=12)
+target_compile_options(rccl PRIVATE -Wuninitialized)
```

Comment:
It might be better to add an extra one `-Wsometimes-uninitialized`.

Reply:
done